Core/Graphics/Source/Graphics.cpp

```diff
@@ -249,6 +252,25 @@ namespace Babylon
         Callback.SetDiagnosticOutput(std::move(outputFunction));
     }
 
+    float Graphics::Impl::GetHardwareScalingLevel()
+    {
+        return m_hardwareScalingLevel;
+    }
+
+    void Graphics::Impl::SetHardwareScalingLevel(float level)
+    {
+        if (level == 0)
```

Comment:
I currently don't like this solution, we should have some "approximately equal to" float comparitor instead.